content/kubermatic/main/architecture/concept/kkp-concepts/applications/application-definition/_index.en.md

```diff
@@ -293,7 +293,17 @@ spec:
 
 ## Advanced Configuration
 ### Default Values
-The `.spec.defaultValues` describe overrides for manifest-rendering in UI when creating an application. (e.g. if the method is Helm, then this field contains the Helm values.).
+The `.spec.defaultValues` describe overrides for manifest-rendering in UI when creating an application. For example if the method is Helm, then this field contains the Helm values.
```

Comment:
These changes should probably also be added to `content/kubermatic/v2.23` (it's added to `main` and `v2.22` in this PR right now).

Reply:
ah the good old "forgot to pull from main" first 🙈  Good point, fixed now :)